CMakeLists.txt

```diff
@@ -128,6 +128,28 @@ option(BUILD_DOCS "Build Api documentation")
 cvc5_option(STATIC_BINARY "Link against static system libraries \
 (enabled by default for static builds)")
 
+# Custom extension for WebAssembly compilation
+set(WASM_FLAGS "" CACHE STRING "Link flags for the WebAssembly binary generation")
+wasm_option(WASM "Use a specific extension for WebAssembly compilation")
+
+# Make sure no other avaible value will be assigned to this variable
+if(NOT(WASM STREQUAL "WASM" OR WASM STREQUAL "JS" OR WASM STREQUAL "HTML"))
+  set(WASM "OFF")
```

Comment:
Instead of silently changing to `OFF`, we should just throw an error. Makes it easier to catch potential typos.